app/locales/pt/locationTracking.json

```diff
@@ -1,12 +1,12 @@
 {
-    "start_logging":"START LOGGING",
-    "stop_logging":"STOP LOGGING",
-    "logging_message":"It is currently logging your location privately every five minutes. Your location information will NOT leave your phone.",
-    "not_logging_message":"NOTE: After clicking this button you may be prompted to grant Private Kit access to your location.",
-    "import":"Import",
-    "export":"Export",
-    "news":"news",
-    "latest_news":"Latest News",
-    "url_info":"For more information visit the Private Kit hompage:",
+    "start_logging":"INICIAR",
+    "stop_logging":"PARAR",
+    "logging_message":"É registrado sua localização atual a cada cinco minutos. Suas informações de localização NÃO sairão do seu telefone.",
```

Comment:
Suggestion for translation: "Sua localização atual está sendo registrada a cada cinco minutos. ..."

Reply:
Thanks friend applied.